buildroot-external/package/intel-e1000e/intel-e1000e.hash

```diff
@@ -0,0 +1,5 @@
+# from https://downloadcenter.intel.com/download/15817/Intel-Network-Adapter-Driver-for-PCIe-Intel-Gigabit-Ethernet-Network-Connections-Under-Linux-
```

Comment:
This URL do not match with the url they are in: https://downloads.sourceforge.net/project/e1000/e1000e%20stable/$(INTEL_E1000E_VERSION)

I think we can remove this part and just hold the local one

Reply:
The URL here is where the official MD5 was located ... someone mirrors the tgz release files on sourceforge. I was not sure what the preferred source would be. We could also switch the URL in the .mk to https://downloadmirror.intel.com/15817/eng/e1000e-3.8.4.tar.gz but I was unsure what you prefer and if intel allows that.
But yeah, I'll go ahead and remove the redundant MD5 as it doesn't provide actual benefit.

Reply:
check